File: Modul_6/Soal_4/PRAK604-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>
#include <string.h>

int main() {
    char kode[100], pesan[100];
    int tagar = 0, bintang = 0;

    scanf("%[^\n]%*c", &kode);
    scanf("%[^\n]%*c", &pesan);

    if(strlen(kode) != strlen(pesan)) {
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    else {
        for(int i = 0; i < strlen(kode); i++) {
            if(kode[i] == ' ' && pesan[i] == ' ') {
                printf(" ");
            }
            else if(kode[i] != pesan[i]) {
                printf("#");
                tagar++;
            }
            else if(kode[i] == pesan[i]){
                printf("*");
                bintang++;
            }
        }
        printf("\n* = %d\n", bintang);
        printf("# = %d\n", tagar);
        if(bintang >= tagar) {
            printf("Pesan Asli");
        }
        else {
            printf("Pesan Palsu");
        }
    }
}

File: Modul_6/Soal_4/PRAK604-2310817210015-MuhammadBukhariFitri.py
kode = input()
pesan = input()
tagar = 0
bintang = 0
if (len(kode) != len(pesan)):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range (len(kode)):
        if(kode[i] == ' ' and pesan[i] == ' '):
            print(" ", end = "")
        elif(kode[i] != pesan[i]):
            print("#", end = "");
            tagar+=1
        elif(kode[i] == pesan[i]):
            print("*", end = "")
            bintang+=1
    print(f"\n* = {bintang}");
    print(f"# = {tagar}");
    if(bintang >= tagar):
        print("Pesan Asli");

    else:
        print("Pesan Palsu");
